src/quadtree.ts

```diff
@@ -464,9 +515,9 @@ export default class QuadTree {
       height,
     };
 
-    // Is the rectangle axis aligned?
-    if (!isAxisAligned(x1, y1, x2, y2))
-      throw new Error("sigma/quadtree.rectangle: shifted view is not yet implemented.");
+    // If the rectangle is shifted, we use the smallest aligned rectangle that contains the shifted one:
+    if (!isRectangleAligned(this.lastRectangle))
```

Comment:
If so, we can have a proper `if-else` to avoid creating the "normal" lastRectangle when camera is shifted.